src/v1/sources/shopify/utils.js

```diff
@@ -8,11 +8,23 @@ const NO_OPERATION_SUCCESS = {
   statusCode: 200,
 };
 
+/**
+ * Updates the anonymousId to userId mapping in Redis
+ * @param {String} anonymousId
+ * @param {String} userId
+ */
+const updateAnonymousIdToUserIdInRedis = async (anonymousId, userId) => {
```

Comment:
moved this to outside util as the userId setting in redis done here is used by both pixel and server side events.